content/v0.75.2.md

```diff
@@ -0,0 +1,129 @@
++++
+title = "0.75.2 Maintenance Release Notes"
+date = 2020-10-25
++++
+
+Installation instructions:
+
+- [Linux](/downloads/linux/) (or [Steam on Linux](/downloads/linux#steam))
+- [Windows](/downloads/windows/)
+- [macOS](/downloads/macos/)
+
+## Table of Contents
+
+- [1. Summary](#1-summary)
+- [2. New Features in Detail](#2-new-features-in-detail)
+  - [2.1. Select Monochrome Palette Colour](#2-1-select-monochrome-palette-colour)
+  - [2.2. Raw Mouse Input](#2-2-raw-mouse-input)
+- [3. Bugfixes](#3-bugfixes)
+- [4. Notes for Packagers](#4-notes-for-packagers)
+- [5. Contributors](#4-contributors)
+- [6. Changelog](#5-changelog)
+
+## 1. Summary
+
+This is a maintenance release from 0.75.x stable branch - it includes a few
+changes and bugfixes backported from upcoming 0.76.x version.
```

Comment:
from **the** 0.75.x 
from **the** upcoming

Reply:
Thanks :)